### PR TITLE
Add QPG6105 IC

### PR DESCRIPTION
**Pull request by lucicop**
Add support for new QPG6105 IC

### CODE REVIEW

File: src/qpg6105/crypto/qpg6100-mbedtls-config.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/*
```
Comment (thcu-gp) [inline src/qpg6105/crypto/qpg6100-mbedtls-config.h:1]: Rename file, still named after qpg6100

File: third_party/Qorvo/CMakeLists.txt [hunk 2]
```diff
@@ -41,7 +41,7 @@ if (OT_MTD)
 endif()
 
 # QPG6100 uses mbedtls_alt
```
Comment (thcu-gp) [inline third_party/Qorvo/CMakeLists.txt:43]: update comment

**Review by thcu-gp — CHANGES_REQUESTED** [review]
Some reference to qpg6100 remaining

**Comment by thcu-gp** [comment]
@jwhui we're adding a new platform, can you review

**Review by jwhui — APPROVED** [review]
Thanks! 👍🏼

**Pull request merged**